core/app/models/concerns/spree/active_storage_adapter/attachment.rb

```diff
@@ -56,15 +58,28 @@ def destroy
       def metadata
         analyze unless analyzed?
 
-        @attachment.metadata
+        @decorated_attachment.metadata
+      end
+
+      def styles_to_transformations(styles)
+        styles.transform_values(&method(:paperclip_to_image_processing_definition))
       end
 
-      def normalize_styles(styles)
-        styles.transform_values { |v| v.split('x').map(&:to_i) }
+      def paperclip_to_image_processing_definition(definition)
```

Comment:
Nitpick: The string is a imagemagick transform string and not a paperclip one. Paperclip just happens to use imagemagick for transfomations.

Reply:
It makes sense. Updated. Thanks.